### ISSUE TITLE
Add an Interval timer mode

### ISSUE THREAD

**Issue by jrmi** [sylvaindubus/workout-chrono#2]
The workout plan is really useful but I also see another usage: an interval timer. In this mode, you have must configure an exercise duration and a rest duration and that's it. When you start the timer it countdown the exercise duration then the rest duration and start again like this video: https://www.youtube.com/watch?v=w_5VVI8esPU&t=1883s but in a configurable way.

Thanks for your app by the way. I did something a bit similar: https://work-move.netlify.app/ I haven't publish the code source yet but may be one day.

**Comment by sylvaindubus** [comment]
I agree, I found it painful sometimes to set a lot of exercises/break sequences. It could definitely be useful, but I won't have the time in the near future to implement that.
Congrats for your Pomodoro btw. 👍
